narwhal/node/src/primary_node.rs

```diff
@@ -325,6 +329,7 @@ impl PrimaryNodeInner {
             store.consensus_store.clone(),
             parameters.gc_depth,
             consensus_metrics.clone(),
+            Self::CONSENSUS_SCHEDULE_CHANGE_SUB_DAGS,
```

Comment:
Is there a reason why we pass this as a parameter instead of making it public?

Reply:
Well mostly for dependency injection reasons